backends-velox/src/test/scala/io/glutenproject/execution/VeloxTPCHSuite.scala

```diff
@@ -81,7 +81,7 @@ abstract class VeloxTPCHSuite extends VeloxTPCHTableSupport {
   def shouldCheckGoldenFiles(): Boolean = {
     Seq("v1", "v1-bhj").contains(subType()) && (formatSparkVersion match {
       case "322" => true
-      case "331" => false
+      case "331" => true
```

Comment:
can we use the major version, `32`, `33` ? what happens if we update Spark minor version

Reply:
Sound good to me, we can match those main version with `startsWith`

Reply:
Update, please take a review.